app/javascript/react/test/boozeYourself.js

```diff
@@ -0,0 +1,19 @@
+import BoozeYourself from '../src/BoozeYourself';
```

Comment:
Your Codeship is failing. This is the error that's showing: `Error: Cannot find module "../src/BoozeYourself"`